### ISSUE TITLE
Testing on training set in supervised learner 08-go-supervised/Go-GPU-run.ipynb

### ISSUE THREAD

**Issue by tommyyliu** [mtrencseni/pytorch-playground#2]
I was trying to reproduce but was getting wildly different results. It was driving me crazy a little bit, then I realized that the code wasn't using the test set. It was using the train set for both.

The test accuracy drops from 75% to about 20% if actually using the test set.

test(model, device, train_loader)
